lib/Formatting/Util.ml

```diff
@@ -0,0 +1,20 @@
+(** [to_element value (module M)] converts [value] to an element
+    using the transforming function provided by [M]. *)
+let to_element
+  : type t. t -> using:(module Interfaces.CONVERTIBLE with type t = t) -> Element.t
+  =
+  fun value ~using:(module M) -> M.to_element value
```

Comment:
I want to say that I quite like this `~using:(module M)` part. It looks nice! I might steal this idea for my use case 👀